.github/workflows/ci.yml

```
@@ -30,12 +30,28 @@ jobs:
 
       - name: Init install
         run: rustup update ${{ matrix.toolchain }} && rustup default ${{ matrix.toolchain }} && rustup component add clippy
-      
+
       - name: Clippy
         run: cargo clippy --all-targets --all-features
 
       - name: Build
         run: cargo build
 
       - name: Tests
-        run: cargo test
\ No newline at end of file
+        run: cargo test
+
+      - name: Install latest nextest release
+        uses: taiki-e/install-action@nextest
+
+      - name: Install cargo-llvm-cov
+        uses: taiki-e/install-action@cargo-llvm-cov
+
+      - name: Collect coverage data
+        run: cargo llvm-cov nextest --lcov --output-path lcov.info
+
+      - name: Report code coverage
+        uses: Nef10/lcov-reporter-action@v0.4.0
+        with:
+          lcov-file: lcov.info
+          pr-number: ${{ env.pr_number }}
```

```suggestion
          pr-number: ${{ github.event.pull_request.number }}
```
I think this can fix the request issue.